bin/quickpkg

```diff
@@ -341,10 +341,6 @@ def quickpkg_main(options, args, eout):
         portage.settings.features.remove("xattr")
         portage.settings.lock()
 
-    if portage.settings.get("BINPKG_GPG_SIGNING_KEY", None):
```

Comment:
I wasn't sure if this was OK because of `unmerge-backup` and `downgrade-backup` but it looks like they just call the entrypoint normally.